src/plugins/data/public/public.api.md

```diff
@@ -78,6 +78,11 @@ import { UnregisterCallback } from 'history';
 import { UnwrapPromiseOrReturn } from '@kbn/utility-types';
 import { UserProvidedValues } from 'src/core/server/types';
 
+// Warning: (ae-missing-release-tag) "ACTION_GLOBAL_APPLY_FILTER" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
```

Comment:
Are these warnings something we should care about?

Reply:
I am not sure. I do not understand the entire auto-generated API thing. Since its only exposing a constant I figured it was good enough.